README.md

```diff
@@ -65,7 +65,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = ">= 3.74.0"
+      version = "~> 3.84"
```

Comment:
This is a relatively old starting version. Any reason not to go with latest 3.X?

Reply:
It was the minimum that supports what we need - do you think it'll exclude customers if we bump to latest?

Reply:
I guess it could exclude air gapped customers if they are unable to get a later version of the provider for any reason. I suspect vast majority would be unaffected. 

Another reason for suggesting the update is bug / security / performance fixes that might have been added since that version. 

But in any case by default it would take the latest version anyway, so not a big deal. Was more curious since we were updating anyway.

Reply:
I have updated to latest now